_pages/clients/python.md

```diff
@@ -64,206 +60,164 @@ client = Algorithmia.client("YOUR_API_KEY", "https://mylocalendpoint")
-
-### Create a Data Collection
+In order to call an Algorithm from Python, we need to first create an algorithm object. With the client already instantiated, we can run the following code to create an object:
 
-This section will show how to create a data collection which is essentially a folder of data files hosted on Algorithmia for free.
+{% highlight python %}
+algo = client.algo('demo/Hello') 
+{% endhighlight %}
 
-First create a data collection called nlp_directory:
+Then, we can use the `.pipe()` method to call the algorithm. We'll provide out input as the argument to the function, and then print the output using the `result` attribute:
 
 {% highlight python %}
-# Instantiate a DataDirectory object, set your data URI and call create
-nlp_directory = client.dir("data://YOUR_USERNAME/nlp_directory")
-# Create your data collection if it does not exist
-if nlp_directory.exists() is False:
-    nlp_directory.create()
+response = algo.pipe("Mr. Bond")
+print(reponse.result)
 {% endhighlight %}
 
-A Data URI uniquely identifies files and directories and contains a protocol "data://" and path "YOUR_USERNAME/data_collection". For more information on the Data URI see the [Data API Specification](http://docs.algorithmia.com/#data-api-specification).
-
-Instead of your username you can also use '.my' when calling algorithms. For more information about the '.my' pseudonym check out the [Hosted Data Guide]({{site.baseurl}}/data/hosted).
-{: .notice-info}
+Which should print the phrase, `Hello Mr. Bond`
 
-### Work with Directory Permissions
+### JSON and Python
 
-When we created the data collection in the previous code snippet, the default setting is `ReadAcl.my_algos` which is a permission type that allows other users on the platform to interact with your data through the algorithms you create if you decide to contribute to algorithm development. This means users can call your algorithm to perform an operation on your data stored in this collection, otherwise the algorithm you created would only work for you.
+The Python client provides some ease of use abstractions for working with alrogithms with JSON inputs and outputs. When passing a Python array or dict into the `.pipe()` function, the library will automatically serialize it to JSON. For algorithms that return a JSON type, the `result` field of the response will contain an array or dict, as appropriate.
 
-To begin working with data directory permissions first add these imports:
+Let's look at an example using JSON and the [WebPredict/ListAnagrams](/algorithms/WebPredict/ListAnagrams) algorithm. The [algorithm docs](https://algorithmia.com/algorithms/WebPredict/ListAnagrams/docs) tell us that the algorithm takes an array of strings and filters it to remove any words that do not have an anagram in the array. We'll call this algorithm using an array of strings as the input:
```

Comment:
Any URLs that refer to specific algorithms should be absolute. (If someone happens to be viewing the documentation in an enterprise setting, the relative path will attempt to load an algorithm that doesn't exist.)

```suggestion
Let's look at an example using JSON and the [WebPredict/ListAnagrams](https://algorithmia.com/algorithms/WebPredict/ListAnagrams) algorithm. The [algorithm docs](https://algorithmia.com/algorithms/WebPredict/ListAnagrams/docs) tell us that the algorithm takes an array of strings and filters it to remove any words that do not have an anagram in the array. We'll call this algorithm using an array of strings as the input:
```

Reply:
We can use [LDA](https://algorithmia.com/algorithms/nlp/LDA/docs) which takes multiple input types such as a file (for working with file section) or text for this section